x-pack/plugins/security_solution/public/overview/components/detection_response/host_alerts_table/host_alerts_table.tsx

```diff
@@ -43,22 +44,28 @@ type GetTableColumns = (
 const DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID = 'vulnerableHostsBySeverityQuery';
 
 export const HostAlertsTable = React.memo(({ signalIndexName }: HostAlertsTableProps) => {
-  const { openTimelineWithFilters } = useNavigateToTimeline();
+  const openAlertsPageWithFilters = useNavigateToAlertsPageWithFilters();
 
-  const openHostInTimeline = useCallback(
-    ({ hostName, severity }: { hostName: string; severity?: string }) => {
-      const hostNameFilter = { field: 'host.name', value: hostName };
-      const severityFilter = severity
-        ? { field: 'kibana.alert.severity', value: severity }
-        : undefined;
+  const openHostInAlerts = useCallback(
+    ({ hostName, severity }: { hostName: string; severity?: string }) =>
+      openAlertsPageWithFilters([
+        {
+          title: 'Host name',
+          selectedOptions: [hostName],
+          fieldName: 'user.name',
```

Comment:
@jamster10 , fieldName should be `host.name` instead of `user.name`?

Reply:
changed! Thanks for catching it